src/components/SpecialResolution/SpecialResolutionForm.vue

```diff
@@ -0,0 +1,463 @@
+  }
+
+  get documentURL (): string {
+    const docUrl = sessionStorage.getItem('BASE_URL') +
+      this.getCreateResolutionResource?.downloadDocPath
+    return docUrl
+  }
+
+  get previewImageSource (): string {
+    // Note: the image file path did not resolve correctly when using the require function directly.  In order
+    // to get the image path resolving correctly, needed to get the image context first and use that to build
+    // the final image file path
+
+    if (this.isJestRunning || !this.getCreateResolutionResource?.previewImagePath) { return '' }
+    const images = require.context('@/assets/images/', false, /\.png$/)
+    return images('./' + this.getCreateResolutionResource?.previewImagePath)
+  }
+
+  /** The name section validity state (when prompted by app). */
+  get invalidNameSection (): boolean {
+    // add more state here
+    return (this.getComponentValidate)
+  }
+
+  /** The minimum date that can be entered (can't be earlier than incorporation date ). */
+  get resolutionDateMinStr (): string {
+    return this.dateToYyyyMmDd(new Date(this.getBusinessFoundingDate))
+  }
+
+  /** The minimum date that can be entered (can't be earlier than incorporation date ). */
+  get resolutionDateMin (): Date {
+    return this.getBusinessFoundingDate
+  }
+  /** The maximum date that can be entered (today). */
+  get resolutionDateMax (): Date {
+    const date = new Date()
+    return date
+  }
+
+  get resolutionTextRules (): Array<Function> {
+    return [
+      v => (v && v.trim().length > 0) || 'Resolution text is required',
+      v => (v && v.length <= this.MAX_RESOLUTION_TEXT_LENGTH) || 'Maximum characters exceeded',
+      v => (v && /^([\w\s$&+,:;=?@#|'<>.^*()%!-\\"]*)$/g.test(v)) || 'Invalid characters'
+    ]
+  }
+
+  /** The maximum date that can be entered (today). */
+  get resolutionDateMaxStr (): string {
+    const date = new Date()
+    return this.dateToYyyyMmDd(date)
+  }
+
+  /** Validations rules for resolution date field. */
+  get resolutionDateRules (): Array<Function> {
+    return [
+      (v: string) => !!v || 'Resolution date is required',
+      (v: string) =>
+
+        this.isBetweenDates(this.resolutionDateMinStr,
+          this.resolutionDateMaxStr,
+          v) ||
+        `Date should be between ${this.dateToPacificDate(this.resolutionDateMin, true)} and
+         ${this.dateToPacificDate(this.resolutionDateMax, true)}`
+    ]
+  }
+
+  /**
+   * True if date is >= the minimum (ie, today) and <= the maximum (ie, the 10th day).
+   * This is used for Vue form validation (in Date Rules above).
+   */
+  // private isBetweenDates (v: string): boolean {
+  isBetweenDates (minDate: string, maxDate: string, dateStrToValidate: string): boolean {
+    if (!dateStrToValidate) { return true }
+    return (new Date(dateStrToValidate) >= new Date(minDate)) && (new Date(dateStrToValidate) <= new Date(maxDate))
```

Comment:
These 3 "new Dates" are actually OK since you're only using them for comparison.

However, I think you could just compare the dates (yyyy-mm-dd), eg.
"is 2022-08-01 < 2022-08-02"? yes
"is 2022-08-01 > 2022-08-02"? no

Reply:
you mean to compare strings?

Reply:
Yes, since they're all yyyy-mm-dd format, right?

Reply:
I leave this up to you.

Reply:
As far as I heard it's better to compare with a date than string.

Reply:
OK.